systers_portal/static/css/style.css

```diff
@@ -781,3 +781,11 @@ li[id=nav-menu]:hover{
     display: block;
   }
 }
+li[id=nav-menu]:hover{
+background-color:#54bceb;
+}
+@media only screen and (min-width: 768px) {
+  .dropdown:hover .dropdown-menu {
+    display: block;
+  }
+}
```

Comment:
@ritwickraj78 why are there CSS and JS changes on this commit if you have a separate commit for frontend?